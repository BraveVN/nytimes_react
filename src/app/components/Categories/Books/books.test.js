
var Books = require('./books');

describe('Books component', () => {
  test('render the Books component', () => {
    const wrapper = shallow(<Books/>);
    expect(wrapper.length).toBe(1);

    var tree = renderer.create(<Books/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
