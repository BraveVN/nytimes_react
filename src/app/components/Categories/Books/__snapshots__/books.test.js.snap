// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Books component render the Books component 1`] = `
<div
  className="category-container"
>
  <h2
    className="title"
  >
    Books
  </h2>
  <div
    className="category-content"
  />
</div>
`;
